src/components/RecentList/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import "./styles.scss";
+import RecentSong from "../RecentSong";
+import data from "./data";
+import ArrowNext from "../Svg/arrowNext";
+function RecentList(props) {
+  const [recentList, setRecentList] = useState(data);
+  return (
+    <div className="recent-list">
+      <div className="recent-list__title">
+        <p className="recent-list__title--left">Gần Đây</p>
+        <a href="#" className="recent-list__title--right">
+          TẤT CẢ
+          <ArrowNext />
+        </a>
+      </div>
+      <div className="recent__lists">
```

Comment:
Đặt tên kế thừa theo class cha nhé
```suggestion
<div className="recent-list__wrapper">
```